auth-web/src/store/modules/user/index.ts

```diff
@@ -0,0 +1,48 @@
+import { Module, VuexModule, Mutation, Action } from 'vuex-module-decorators'
+import keycloakService from '@/services/keycloak.services'
+import userServices from '@/services/user.services'
+import { UserInfo } from '@/models/userInfo'
+
+@Module({
+  name: 'user'
+})
+export default class UserModule extends VuexModule {
+  currentUser: UserInfo
+
+  userProfile: any
+
+  @Mutation
+  public setUserProfile (userProfile: any) {
```

Comment:
create a model for userprofile and use it here instead of any?
need not to do immediately

Reply:
That's the plan, will do it with user profile ticket